auth/src/main/java/org/aerogear/auth/AbstractAuthenticator.java

```diff
@@ -1,6 +1,7 @@
 package org.aerogear.auth;
```

Comment:
packages need to be renamed to conform to https://github.com/aerogear/proposals/blob/master/sdks/SDK-naming.md

Reply:
@secondsun : just to be sure, should we rename it to `org.aerogear.android.ags.auth`?

Reply:
@ziccardi Yes